modules/common/hardware/definition.nix

```diff
@@ -63,6 +63,24 @@
       };
     };
 
+    disks = mkOption {
```

Comment:
I was a bit surprised that hardware definitions can only set a subset instead of all nixos options, but I went along with it for now.